config/locales/en.yml

```diff
@@ -108,8 +108,8 @@ en:
     address: Address
     advancement:
       good_candidate: Good candidate
-      irrelevant: Completely irrelevant
-      might_be_irrelevant: Might be irrelevant
+      irrelevant_candidate: Completely irrelevant
+      relevant_candidate: Might be irrelevant
```

Comment:
looks strange, may we rename `relevant_candidate` to `might_be_irrelevant_candidate` or smth like this?

Reply:
In the database, this score is named relevant